packages/ag-charts-community/src/chart/axis/axisInterval.ts

```diff
@@ -3,7 +3,7 @@ import type { TickInterval } from './axisTick';
 
 export class AxisInterval<S> extends BaseProperties {
     @Property
-    placement?: 'on' | 'between';
+    placement?: 'on' | 'between' = 'between';
```

Comment:
```suggestion
    placement: 'on' | 'between' = 'between';
```

Reply:
I tried applying this change, but it causes typescript compilation issues in linearGaugeSeries.ts which can't be easily worked around.